### DISCUSSION TITLE
Allow rr get without generating a config file sample

### DISCUSSION THREAD

**Discussion by acelaya** [roadrunner-server/roadrunner#1287] [Ideas]
### I have an idea!

I have a project where roadrunner's config already exists in a different folder than the one where the rr binary is downloaded, and I pass it t with `-c path/to/.rr.yml` when starting the server.

Would you be open to add a flag (let's say `--no-config`, or something in those lines) to the `./vendor/bin/rr get` command to not generate a config file?

If so, I'm open to contribute this.

**Comment by acelaya** [comment]
I just realized that tool actually lives in https://github.com/spiral/roadrunner-cli, so I guess this issue should be transferred there.

**Reply by rustatian** [reply]
Hey @acelaya 👋🏻 . Yeah, but I can't move this discussion/issue to a different organization. You'll need to recreate the issue manually in the proper repo. Thanks.

**Reply by acelaya** [reply]
Done https://github.com/spiral/roadrunner-cli/issues/37
